_build_cfg.yml

```diff
@@ -191,8 +191,8 @@ Topics:
     File: application_health
   - Name: Events
     File: events
-  - Name: Output Build Dependencies
-    File: output_deps
+  - Name: Dependency Tree Generation
+    File: dependecy_tree_generation
```

Comment:
Typo here and in the actual `dev_guide/dependecy_tree_generation.adoc` file name itself.